lib/errors.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+const errorTitle = (code) => `${code}_ERROR`;
+const errorMsg = (code) => `There is a ${code}_ERROR`;
+
+const getContent = (err, debug, translate) => {
+  let content = {};
```

Comment:
But the content of the `{}` will change, isn't that the same as `content` changing?